sdk/metricsadvisor/ai-metrics-advisor/test/public/util/matrix.ts

```diff
@@ -1,48 +0,0 @@
-// Copyright (c) Microsoft Corporation.
```

Comment:
is this removal intentional?

Reply:
Yes, this file defines `matrix`, the test utility function already exported by `@azure/test-utils`, so I imported it from the latter instead.

Reply:
oh excellent! I love reuse